partner_duns/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Partner DUNS",
     "summary": "Set DUNS (Data Universal Numbering System) on partners",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.0.1",
```

Comment:
There's no need to manually bump versions. Our merge bot will take care of it for us 😉